tests/nix_tests/ibc_utils.py

```diff
@@ -1,116 +1,118 @@
 import json
 import subprocess
 from pathlib import Path
-from typing import NamedTuple
+from typing import Any, Dict, List, NamedTuple
 
 from pystarport import ports
 
-from .network import CosmosChain, Evmos, Hermes, setup_custom_evmos
+from .network import CosmosChain, Hermes, setup_custom_evmos
 from .utils import ADDRS, eth_to_bech32, wait_for_port
 
 # EVMOS_IBC_DENOM IBC denom of aevmos in crypto-org-chain
 EVMOS_IBC_DENOM = "ibc/8EAC8061F4499F03D2D1419A3E73D346289AE9DB89CAB1486B72539572B1915E"
 RATIO = 10**10
+# IBC_CHAINS_META metadata of cosmos chains to setup these for IBC tests
+IBC_CHAINS_META = {
+    "chainmain": {
+        "chain_name": "chainmain-1",
+        "bin": "chain-maind",
+        "denom": "basecro",
+    },
+    "stride": {
+        "chain_name": "stride-1",
+        "bin": "strided",
+        "denom": "ustrd",
+    },
+    "osmosis": {
+        "chain_name": "osmosis-1",
+        "bin": "osmosisd",
+        "denom": "uosmo",
+    },
+    "gaia": {
+        "chain_name": "cosmoshub-1",
+        "bin": "gaiad",
+        "denom": "uatom",
+    },
+}
 
 
 class IBCNetwork(NamedTuple):
-    evmos: Evmos
-    other_chain: CosmosChain
+    chains: Dict[str, Any]
     hermes: Hermes
-    incentivized: bool
 
 
-def prepare_network(tmp_path, file, other_chain_name, incentivized=False):
+def prepare_network(tmp_path, file, other_chains_names: List[str]):
```

Comment:
It's interesting to me that we're using type hints in some places but ditch them in most places
```suggestion
def prepare_network(tmp_path: Path, file: str, other_chains_names: List[str]):
```